error_check_logic.py

```diff
@@ -64,7 +66,7 @@ def RunUpgradeScript(self):
     groups_students = models.GroupsStudents.Fetch(self.institution, self.session)
     models.GroupsStudents.store(self.institution, self.session, groups_students)
 
-    models.DBVersion.Store(self.institution, self.session, CURRENT_DB_VERSION)
+    self.setDBVersion(self.institution, self.session, CURRENT_DB_VERSION)
```

Comment:
Oops! But why is it working?
OK, I just learned something from docs.python:

> [A class method] can be called either on the class (such as C.f()) or on an instance (such as C().f()). The instance is ignored except for its class.

Leaving it in. Bleh.